app/src/androidTest/java/com/sdp/movemeet/view/map/MainMapFragmentTest.java

```diff
@@ -164,7 +164,7 @@ public void cameraZoomsOnlyOnFirstCall() {
     public void clickingOnActivityMakesCorrectIntentWhenLogged() {
         MainMapFragment mapFragment = fragmentTestRule.getFragment();
         Activity act = new Activity("activity id", "organizer id", "my title", 4, new ArrayList<>(), 0, 0,
-                    "desc", null, new Date(), 1, Soccer, "Dubai UAE", new Date());
+                    "desc", "activities/123455", new Date(), 1, Soccer, "Dubai UAE", new Date());
```

Comment:
As discussed with Victor finally, even if this won't change anything in the end (because this activity object here is not uploaded to Firebase Firestore), it's a very good practice to have changed the documentPath like this! In fact, the documentPath of activities in Firebase Firestore are always of the form "activities/<some-string>".

Reply:
Yes !